build.py

```diff
@@ -64,14 +64,18 @@
 		'-I/usr/include/libxml2'
 	]
 
-# Disable until changes are merged into dispatch.
-#if "LIBDISPATCH_SOURCE_DIR" in Configuration.current.variables:
-#        foundation.CFLAGS += " "+" ".join([
-#                '-DDEPLOYMENT_ENABLE_LIBDISPATCH',
-#                '-I'+Configuration.current.variables["LIBDISPATCH_SOURCE_DIR"],
-#                '-I'+Configuration.current.variables["LIBDISPATCH_BUILD_DIR"]+'/tests'  # for include of dispatch/private.h in CF
-#        ])
-
+# Configure use of Dispatch in CoreFoundation and Foundation if libdispatch is being built
+if "LIBDISPATCH_SOURCE_DIR" in Configuration.current.variables:
```

Comment:
Thanks for picking that up. Looks like I missed a line when I re-commented out the block as building Dispatch into Foundation currently hits compile errors in NSOperation (due to some Dispatch overlay work that needs fixing up).